### ISSUE TITLE
Error: Cannot find module './globals.js' - react-spring/dist/web.cjs.js:8:1

### ISSUE THREAD

**Issue by stolinski** [drcmda/react-spring#170]
In the latest version 5.5.2 I'm getting 

```
W20180803-12:30:54.576(-6)? (STDERR) Error: Cannot find module './globals.js'
W20180803-12:30:54.576(-6)? (STDERR)     at Function.Module._resolveFilename (module.js:547:15)
W20180803-12:30:54.576(-6)? (STDERR)     at Function.Module._load (module.js:474:25)
W20180803-12:30:54.576(-6)? (STDERR)     at Module.require (module.js:596:17)
W20180803-12:30:54.576(-6)? (STDERR)     at require (internal/module.js:11:18)
W20180803-12:30:54.576(-6)? (STDERR)     at Object.<anonymous> (/Users/scotttolinski/Sites/levelup/node_modules/react-spring/dist/web.cjs.js:8:1)
W20180803-12:30:54.576(-6)? (STDERR)     at Module._compile (module.js:652:30)
W20180803-12:30:54.577(-6)? (STDERR)     at Object.Module._extensions..js (module.js:663:10)
W20180803-12:30:54.577(-6)? (STDERR)     at Module.load (module.js:565:32)
W20180803-12:30:54.577(-6)? (STDERR)     at tryModuleLoad (module.js:505:12)
W20180803-12:30:54.577(-6)? (STDERR)     at Function.Module._load (module.js:497:3)
W20180803-12:30:54.577(-6)? (STDERR)     at Module.require (module.js:596:17)
W20180803-12:30:54.580(-6)? (STDERR)     at require (internal/module.js:11:18)
W20180803-12:30:54.580(-6)? (STDERR)     at npmRequire (/Users/scotttolinski/Sites/levelup/.meteor/local/build/programs/server/npm-require.js:133:10)
W20180803-12:30:54.580(-6)? (STDERR)     at Module.useNode (packages/modules-runtime.js:699:18)
W20180803-12:30:54.581(-6)? (STDERR)     at web.cjs.js (packages/modules.js:786:8)
W20180803-12:30:54.581(-6)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:339:7)
W20180803-12:30:54.581(-6)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180803-12:30:54.581(-6)? (STDERR)     at Modals.js (imports/ui/elements/Modals.js:1:264)
W20180803-12:30:54.581(-6)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:339:7)
W20180803-12:30:54.582(-6)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180803-12:30:54.624(-6)? (STDERR)     at index.js (imports/ui/elements/index.js:1:71)
W20180803-12:30:54.624(-6)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:339:7)
W20180803-12:30:54.624(-6)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180803-12:30:54.625(-6)? (STDERR)     at Header.js (imports/ui/layouts/Header.js:1:513)
W20180803-12:30:54.625(-6)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:339:7)
W20180803-12:30:54.625(-6)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180803-12:30:54.625(-6)? (STDERR)     at App.js (imports/ui/layouts/App.js:1:633)
W20180803-12:30:54.625(-6)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:339:7)
W20180803-12:30:54.625(-6)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180803-12:30:54.635(-6)? (STDERR)     at ssr-init.js (imports/startup/server/ssr-init.js:1:1274)
W20180803-12:30:54.635(-6)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:339:7)
W20180803-12:30:54.636(-6)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180803-12:30:54.636(-6)? (STDERR)     at index.js (imports/startup/server/index.js:1:218)
W20180803-12:30:54.636(-6)? (STDERR)     at fileEvaluate (packages/modules-runtime.js:339:7)
W20180803-12:30:54.636(-6)? (STDERR)     at require (packages/modules-runtime.js:238:16)
W20180803-12:30:54.636(-6)? (STDERR)     at /Users/scotttolinski/Sites/levelup/.meteor/local/build/programs/server/app/app.js:28499:15
W20180803-12:30:54.636(-6)? (STDERR)     at /Users/scotttolinski/Sites/levelup/.meteor/local/build/programs/server/boot.js:411:36
W20180803-12:30:54.680(-6)? (STDERR)     at Array.forEach (<anonymous>)
W20180803-12:30:54.680(-6)? (STDERR)     at /Users/scotttolinski/Sites/levelup/.meteor/local/build/programs/server/boot.js:220:19
W20180803-12:30:54.680(-6)? (STDERR)     at /Users/scotttolinski/Sites/levelup/.meteor/local/build/programs/server/boot.js:471:5
W20180803-12:30:54.680(-6)? (STDERR)     at Function.run (/Users/scotttolinski/Sites/levelup/.meteor/local/build/programs/server/profile.js:510:12)
W20180803-12:30:54.681(-6)? (STDERR)     at /Users/scotttolinski/Sites/levelup/.meteor/local/build/programs/server/boot.js:470:11
```

**Comment by drcmda** [comment]
@stolinski should be fixed in 5.5.4, i tried to build this lib with a windows computer, "/" made rollup externalize all files. :-S

**Comment by stolinski** [comment]
Thanks!
